ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+
+#Employee Total
+puts "How many employees"
+employee_total= gets.chomp.to_i
+#Loop 
+employee=0
+until employee == employee_total
+
+#Name
+puts "what's your name"
```

Comment:
When we write code we need its structure to be easy to navigate.
Our best tools are line spacing and indentation; we use line spacing to separate our code into logical block and use appropriate indentation(2 spaces) to show how our code is related. Check out the airbnb Ruby style guide for some great examples on how to indent properly. https://github.com/airbnb/ruby